types.go

```diff
@@ -79,15 +79,16 @@ func IsAWSARN(target string) bool {
 // IsDockerImage returns true if the target is a Docker image.
 //
 // The registry must be specified, while the tag is optional:
-//   Valid: registry.hub.docker.com/metasploitframework/metasploit-framework:latest
-//   Valid: registry.hub.docker.com/metasploitframework/metasploit-framework
-//   Valid: registry.hub.docker.com/library/debian
-//   Valid: registry.hub.docker.com/path1/path2/artifact (compliant with V2 spec)
-//   Valid: registry.hub.docker.com/artifact (compliant with V2 spec)
-//   Valid: localhost:5500/library/debian
-//   Not valid: metasploitframework/metasploit-framework:latest
-//   Not valid: metasploitframework/metasploit-framework
-//   Not valid: debian
+//
+//   - Valid: registry.hub.docker.com/metasploitframework/metasploit-framework:latest
+//   - Valid: registry.hub.docker.com/metasploitframework/metasploit-framework
+//   - Valid: registry.hub.docker.com/library/debian
```

Comment:
I would add an example with `docker.io` and perhaps some with other big registries like quay/ghcr.io, just to make it clear.

Reply:
I have added the examples and tests in commit 7222b2ab44ff1e7d6ea4f55d615c3a8c374a7353